.semaphore/semaphore.yml

```diff
@@ -4,7 +4,7 @@ name: "Vaultenv CI Pipeline"
 
 agent:
   machine:
-    type: "e1-standard-4"
+    type: "f1-standard-4"
```

Comment:
Shouldn't this be

```suggestion
    type: "f1-standard-2"
```

Reply:
The idea was to keep the same amount of cores per run. But if you think 2 cores should be enough I'll ship that

Reply:
Then I misexplained, my intention (and Semaphore's recommendation) was to substitute 1 f1 core for every 2 e2 cores, on the grounds that the f1 cores still outperformed at that conversion rate.